test/spec/adapters/wideorbit_spec.js

```diff
@@ -489,6 +489,25 @@ describe('wideorbit adapter tests', function () {
             stubAddBidResponse.restore();
         });
 
+        it('should do nothing when type is set to unrecognized type', function () {
+
+            var stubAddBidResponse = sinon.stub(bidmanager, 'addBidResponse');
+
+            var response = {
+                UserMatchings: [
+                    {
+                        Type: 'unrecognized',
+                        Url: 'http%3A%2F%2Fwww.admeta.com%2F1.js'
+                    }
+                ],
+                Placements: placements
+            };
+
+            $$PREBID_GLOBAL$$.handleWideOrbitCallback(response);
+
```

Comment:
It will actually call addBidResponse. It will however not throw an exception as it did before. I agree the test isn't very good, but it did catch my particular issue.